frontend/Cargo.toml

```diff
@@ -9,10 +9,8 @@ license = "MIT OR Apache-2.0"
 keywords = ["yew", "trunk"]
 categories = ["gui", "wasm", "web-programming"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
 yew = { version="0.20", features=["csr"] }
 yew-router = "0.15.0"
-yew_styles = { version="0.11", features=["layouts", "button"] }
```

Comment:
This turned out to be a pretty poor library. I did [find this](https://github.com/yewstack/yewtify) though, which is very similar to the framework I used for my website. Probably something I'll play around with. It's still pretty young.